### PR TITLE
Feature/phantom

### PR DESCRIPTION
**Pull request by josefbaltz**
### Information

phantom

### CODE REVIEW

**Review by tonyswink — APPROVED** [review]
LGTM; Approved.

File: src/main/java/tv/galaxe/genesis/Core.java [hunk 1]
```diff
@@ -16,11 +16,12 @@ public void onEnable() {
...
 
 		// Commands
-		// getCommand("genesis").setExecutor(new GenesisGUI());
+		getCommand("genesis").setExecutor(new SelectGUI());
 
 		// Genus Enforcers
-		getServer().getPluginManager().registerEvents(new Skeleton(), this);
-		getServer().getPluginManager().registerEvents(new Enderman(), this);
+		// getServer().getPluginManager().registerEvents(new Skeleton(), this);
+		// getServer().getPluginManager().registerEvents(new Enderman(), this);
```
Comment (garrettsummerfi3ld) [inline src/main/java/tv/galaxe/genesis/Core.java:23]: Reason why this is commented out?

**Review by garrettsummerfi3ld — APPROVED** [review]
LGTM

File: src/main/java/tv/galaxe/genesis/Core.java (hunk 1 above)
Comment (josefbaltz) [reply src/main/java/tv/galaxe/genesis/Core.java:23]: Need to implement some kind of staff permission so it doesn't apply all class types to users with * permission

**Pull request merged**